tests/v1/tpu/test_basic.py

```diff
@@ -63,3 +63,40 @@ def test_basic(
         output = vllm_outputs[0][1]
 
         assert "1024" in output or "0, 1" in output
+
+
+@pytest.mark.skipif(not current_platform.is_tpu(),
+                    reason="This is a basic test for TPU only")
+def test_gemma3_with_mm_on_multichip(
```

Comment:
The test name indicates there's MM, could you help point out where's it?

Reply:
updated